api/config.ts

```diff
@@ -0,0 +1,15 @@
+import { Client, Account} from "appwrite";
+
+export const appwriteConfig = {
+     url: "https://cloud.appwrite.io/v1",
+     projectId: "6616ea581ef9f5521c7d",
```

Comment:
Move this to a .env.local file and change to process.env variable

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

Reply:
Per our convo with Shashi this afternoon, he mentioned we can hardcode it at the moment :)